scripts/cover.sh

```diff
@@ -128,7 +128,7 @@ cat ./coverage/istanbul.json | jq '[
 ] | from_entries' > ./coverage/istanbul-runtime.json
 
 echo "Checking code coverage for runtime folder"
-./node_modules/.bin/istanbul check-coverage --statements 100 \
+./node_modules/.bin/istanbul check-coverage --statements 99.92 \
```

Comment:
do we want to lower this from 100?

Reply:
there were cases in code that were not covered and these flows I didn't touch. But since I touched the tchannel_client.go file the build started failing. Also, those cases are erring cases in the workflow they are part of while ini'ting, which is not super important. Hence lowering the threshold on discussion with @jacobg.